File: task5.js
// Move all the negative elements to one side of the array 

function NegativeToOneSide(numbers) {
    let p = new Array(0)
    let n = new Array(0)

    numbers.map((edx) => {
        if (edx >= 0)
            p.push(edx)
        else
            n.push(edx)
    })

    return [...p, ...n]
}

let arr = [3, 53, -43, -54, 73, 4, -3, -64, -21, 9]

console.log(NegativeToOneSide(arr))

File: task5.py
# Move all the negative elements to one side of the array

def moveNegatives(nums):
    n = []
    p = []
    for i in nums:
        if (i < 0):
            n.append(i)
        else:
            p.append(i)

    # print(n, "---", p)

    return n+p


numbers = [23, 42, 34, -23, -423, 4, 23, -4, ]

print(moveNegatives(numbers))
